test/end-to-end/core.sh

```diff
@@ -464,7 +464,6 @@ echo "[INFO] Validating routed app response..."
 # will be reachable via the ip of its pod.
 router_ip=$(oc get pod "${router_pod}" --template='{{.status.podIP}}')
 CONTAINER_ACCESSIBLE_API_HOST="${CONTAINER_ACCESSIBLE_API_HOST:-${router_ip}}"
-validate_response "-s -k --resolve www.example.com:443:${CONTAINER_ACCESSIBLE_API_HOST} https://www.example.com" "Hello from OpenShift" 0.2 50
```

Comment:
Have no idea - but it looked like it was expecting some remanent of a route to exist. The right test would be to create the route and see that it works - which is what the `code` that follows does.